scripts/update/Updater.php

```diff
@@ -1904,5 +1904,13 @@ public function update($initialVersion) {
         }
 
         $this->skip('34.0.0', '34.2.0');
+
+        if ($this->isVersion('34.2.0')) {
+            $this->getServiceManager()->register(
+                TimerStrategyService::SERVICE_ID,
```

Comment:
still not sure that we need it as a full service, maybe timer strategy provider would be enough?